src/pages/demos/navigation.hbs

```diff
@@ -0,0 +1,430 @@
+    <article class="mzp-c-article">
+      <h1 class="mzp-c-article-title">Navigation Demo</h1>
+
+      <p>Et pri porro ceteros tacimates, est at option dignissim intellegat.
+        Tollit fuisset urbanitas nec no, feugiat sensibus an mea, nominavi
+        <a href="#">interpretaris te eam</a>. No virtute imperdiet pro, ut pro
+        sint viderer philosophia. Torquatos necessitatibus sit in, ad sit summo
+        molestie adolescens, liber oratio veniam in per.</p>
+
+      <p>No qui option timeam impetus, dolor salutatus pertinacia ea eam, at
+        inani summo democritum vis. Est id quis admodum tractatos, ei ius
+        eligendi platonem, maiorum eleifend et eam. Ei odio cetero voluptatibus
+        ius, prima blandit epicurei te cum. Illud verterem principes vix te,
+        te pro tale dicit. Debitis senserit no quo, vix et altera mollis
+        scaevola. Sit no soleat aperiri vituperata. Qui <a href="#">id placerat
+        euripidis</a>.</p>
+
+      <h2>Tale augue splendide in quo</h2>
+
+      <ul class="mzp-u-list-styled">
+        <li>Diam repudiare vel cu.</li>
+        <li>Meis augue regione mei at, <a href="#">ornatus epicurei</a> ius eu.</li>
+        <li>Nam nemore consulatu et, alia hinc ex pro. Sumo rebum ea his, cu
+          possit nostro interesset has. Vide mollis eleifend cum cu, nec an
+          aeque simul concludaturque.</li>
+        <li>Eius omnium appellantur eam ne, pri an unum appellantur. Cu ius tota
+          posse dicant, autem quodsi evertitur eum no.</li>
+      </ul>
+
+      <h3>Affert graeco eum id</h3>
+
+      <p>An quo epicurei placerat repudiare. Ut cetero molestiae mei, per id
+        efficiantur consectetuer, in paulo repudiare vim. Inermis delectus
+        adipiscing eos ad, sed ei graeci eleifend intellegam. Eu abhorreant
+        neglegentur duo, eu mei nisl mundi.</p>
+
+      <ol class="mzp-u-list-styled">
+        <li>Veniam saperet corrumpit in has. Vim movet doming concludaturque an,
+          elitr libris sit ei. Ut vel probo viris, qui persius scripserit
+          repudiandae ut.</li>
+        <li>Eum doming forensibus at. Has an justo aperiam evertitur. Sea dolor
+          nullam scripta ea, possit appareat pericula ea pro.</li>
+        <li>Mei sint consequuntur ei, est ut quando omittam, possim efficiendi sed id.</li>
+      </ol>
+
+      <p>Suavitate comprehensam ea nam, tacimates vituperata vim te. Usu in
+        laboramus maiestatis definitionem, noster complectitur at vim. Qui ut
+        doctus sententiae percipitur, vivendum mediocritatem ea eum. Eu cum
+        tantas nostrud, ea docendi blandit nec, vis id aeterno intellegat
+        accommodare.</p>
+    </article>
+  </main>
+</div>
+
+<script src="{{@root.baseurl}}/assets/protocol/protocol/js/protocol-menu.js"></script>
+<script src="{{@root.baseurl}}/assets/protocol/protocol/js/protocol-navigation.js"></script>
+
+<script>
+(function() {
+    'use strict';
+
+    Mozilla.Navigation.init({
```

Comment:
@stephaniehobson this is where init gets called in the demo page. It’s also detailed in the docs for both components.

Reply:
Oooooooh. That explains why I coulldn't find it in the .js files >_<